src/main/kotlin/com/asap/asapbackend/domain/todo/domain/service/TodoRemover.kt

```diff
@@ -8,9 +8,17 @@ import org.springframework.stereotype.Service
 class TodoRemover(
     private val todoRepository: TodoRepository
 ) {
-    fun delete(todo: Todo){
+    fun delete(todo: Todo) {
         todo.let {
             todoRepository.delete(it)
         }
```

Comment:
이거 내가 만든거긴한데 바꿔주라..ㅎ
```suggestion
        todoRepository.delete(it)
```